manticore/core/manticore.py

```diff
@@ -428,6 +428,16 @@ def clear_terminated_states(self):
             self._remove(state_id)
         assert self.count_terminated_states() == 0
 
+    @sync
+    @at_not_running
+    def clear_ready_states(self):
```

Comment:
This function is called `clear_ready_states`, but it's a duplicate of `clear_terminated_states` just above it.